pkg/webhooks/jobset_webhook.go

```diff
@@ -382,7 +382,7 @@ func validateCoordinator(js *jobset.JobSet) error {
 
 // replicatedJobByName fetches the replicatedJob spec from the JobSet by name.
 // Returns nil if no replicatedJob with the given name exists.
-func replicatedJobByName(js *jobset.JobSet, replicatedJob string) *jobset.ReplicatedJob {
+func replicatedJobByName(js *jobset.JobSet) *jobset.ReplicatedJob {
```

Comment:
Looks like in https://github.com/kubernetes-sigs/jobset/pull/627 the `replicatedJobByName` function had a bug that was missed. We should keep the `replicatedJob` string param for this function, but on line 387 it should say:

```go
		if rjob.Name == replicatedJob { 
                    ...
                 }

```

rather than hardcoding the rjob.Name == js.spec.coordinator.replicatedJob. The coordinator replicatedJob is being passed in as a param by the caller.

Reply:
kk. Feel free to propose a PR to fix that bug. I'll add this in after.

Reply:
sounds good, just created #645 with the fix

Reply:
updated with your fix.